server/core/store.go

```diff
@@ -302,24 +302,33 @@ func (s *StoreInfo) regionScoreV1(highSpaceRatio, lowSpaceRatio float64, delta i
 }
 
 func (s *StoreInfo) regionScoreV2(delta int64, deviation int) float64 {
+	var amplification float64
 	A := float64(float64(s.GetAvgAvailable())-float64(deviation)*float64(s.GetAvailableDeviation())) / gb
 	C := float64(s.GetCapacity()) / gb
-	R := float64(s.GetRegionSize() + delta)
 	var (
 		K, M float64 = 1, 256 // Experience value to control the weight of the available influence on score
 		F    float64 = 20     // Experience value to prevent some nodes from running out of disk space prematurely.
 	)
+	used := float64(s.GetUsedSize()) / mb
```

Comment:
Duplicate with `Available`?

Reply:
No, `A !=  C-U`, the used size is the `datadir` size of the tikv, but availabel is the remaining space of the disk.